src/NLog/LogFactory.cs

```diff
@@ -114,6 +114,7 @@ public LogFactory()
         /// Initializes a new instance of the <see cref="LogFactory" /> class.
         /// </summary>
         /// <param name="config">The config.</param>
+        [Obsolete("Constructor with LoggingConfiguration as parameter should not be used. Instead provide LogFactory as parameter when constructing LoggingConfiguration.")]
```

Comment:
add the text, obsolete in NLog 5? (or something like that)

Reply:
Fixed